File: server-yii/models/Article.php
<?php

namespace app\models;

use Yii;

//
// Created by zhanxiaoping 
// zhanxp@me.com
//
class Article extends Base 
{
    // public $title;
    // public $intro;
    // public $picUrl;
    // public $content;
    // public $categoryId;

    public static function tableName()
    {
        return 'article';
    }
}

File: server-spring-boot/enuocms-business/enuocms-business.api/src/main/java/com/enuocms/business/article/Article.java
package com.enuocms.business.article;

import com.enuocms.core.model.BaseEntity;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.ToString;

import javax.persistence.Table;

/**
 * Created by zhanxiaoping on 2017/8/28.
 * article
 * zhanxp@me.com
 */
@Data
@EqualsAndHashCode(callSuper = true)
@ToString
@Table(name = "article")
public class Article extends BaseEntity {
    private String title;

    private String intro;

    private String picUrl;

    private Integer categoryId;

    private String content;
}
